types/stream-meter/index.d.ts

```diff
@@ -1,15 +1,46 @@
 /// <reference types="node" />
 
-import { Transform } from "stream";
-
-declare function m(maxBytes?: number): m.StreamMeter;
+import { Transform } from "node:stream";
```

Comment:
This is technically a breaking change for anyone using an older version of the node types, not that it's a big deal.

Reply:
Done.